aws_marketplace/using_algorithms/automl/AutoML_-_Train_multiple_models_in_parallel.ipynb

```diff
@@ -0,0 +1,635 @@
+    "6.4          | 3.2         | 4.5          | 1.5         | versicolor\n",
+    "6.9          | 3.1         | 4.9          | 1.5         | versicolor\n",
+    "&hellip;     | &hellip;    | &hellip;     | &hellip;    | &hellip;\n",
+    "6.5          | 3.0         | 5.2          | 2.0         | virginica\n",
+    "6.2          | 3.4         | 5.4          | 2.3         | virginica\n",
+    "5.9          | 3.0         | 5.1          | 1.8         | virginica\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "training_data_location='data/training/iris.csv'\n",
+    "urllib.request.urlretrieve('https://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data',training_data_location)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us look at the sample training data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!head $training_data_location"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us add a header and a copy of first line to demonstrate that the AutoML listing takes care of missing values as well."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!sed -i '1s/^/sepal_length,sepal_width,petal_length,petal_width,species\\n,,1.4,0.2,Iris-setosa\\n/' $training_data_location"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!head $training_data_location"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When training large models with huge amounts of data, you'll typically use big data tools, like Amazon Athena, AWS Glue, or Amazon EMR, to create your data in S3. For the purposes of this example, we're using the classic [Iris dataset](https://en.wikipedia.org/wiki/Iris_flower_data_set), which the notebook downloads from the source. \n",
+    "\n",
+    "We can use use the tools provided by the Amazon SageMaker Python SDK to upload the data to an S3 bucket. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training_input = sagemaker_session.upload_data(training_data_location, bucket, key_prefix=training_input_prefix)\n",
+    "print (\"Training Data Location \" + training_input)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 4: Train a model\n",
+    "Next, let us train a model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "algo = AlgorithmEstimator( \n",
+    "algorithm_arn=algorithm_arn, \n",
```

Comment:
Please indent this code inside the function call appropriately to make it more readable

Reply:
incorporated.